master/docs/manual/cfg-www.rst

```diff
@@ -571,12 +574,9 @@ Here is a configuration that is known to work (Apache 2.4.10 / Debian 8), direct
         ServerName buildbot.example
         ServerAdmin webmaster@buildbot.example
 
-        <Location /ws>
-          ProxyPass ws://127.0.0.1:8020/ws
-          ProxyPassReverse ws://127.0.0.1:8020/ws
-        </Location>
-
-        ProxyPass /ws !
+        # replace with actual port of your Buildbot master
+        ProxyPass ws://127.0.0.1:8020/ws
```

Comment:
@gracinet hey, are you kidding me? :D Well, you use 
```
ProxyPass /ws ws://localhost:8011/ws
ProxyPassReverse /ws ws://localhost:8011/ws
```
which *is same as*
```
<Location /ws>
         ProxyPass ws://127.0.0.1:8011/ws
         ProxyPassReverse ws://127.0.0.1:8011/ws
</Location>
```
but without `<Location /ws>`, or without `ProxyPass /ws` it would be broken. So, in PR you just remove `Location` but didn't add ` /ws`, which you are using actually :) So, original config in docs was correct.